flink-state-backends/flink-statebackend-changelog/src/main/java/org/apache/flink/state/changelog/ChangelogKeyGroupedPriorityQueue.java

```diff
@@ -133,11 +133,25 @@ public StateChangeApplier getChangeApplier(ChangelogApplierFactory factory) {
         return factory.forPriorityQueue(delegatedPriorityQueue, serializer);
     }
 
+    @Override
+    @SuppressWarnings("unchecked")
+    public <IS> void setDelegatedState(IS state) {
+        this.delegatedPriorityQueue = (KeyGroupedInternalPriorityQueue<T>) checkNotNull(state);
+    }
+
+    public StateChangeLogger<T, Void> getStateChangeLogger() {
+        return logger;
+    }
+
     @Override
     public void resetWritingMetaFlag() {
         logger.resetWritingMetaFlag();
     }
 
+    public TypeSerializer<T> getSerializer() {
+        return serializer;
+    }
+
```

Comment:
I couldn't find any usages of this method.

Reply:
removed